blocks/footer/footer.css

```diff
@@ -1142,6 +1142,12 @@
   }
 }
 
+@media (min-width: 80rem) {
+  .footer-legal-links ul {
+    margin: unset;
```

Comment:
@bosschaert , I think the `margin` for `ul` styles are coming from [this commit](https://github.com/hlxsites/zeiss/commit/f47c61fffe41e50e40bc54712f44d7f70b500743). Can we make the selector more specific there if it applies only for specific situations? If yes, we'd not need to override it here and elsewhere. Of course if that style is globally needed, we'd need to override it here.

Reply:
We could use a more specific margin for `ul` styles for `.text` blocks:

```
.text ul {
  margin: var(--spacing-vertical-m) 0;
}
```